traffic-control-openshift/README.md

```diff
@@ -1,25 +1,61 @@
 # Traffic Control on OpenShift
 This demo demonstrates how to use traffic control to test applications running in [OpenShift](https://www.openshift.com/). It uses OpenShift version 3 to run the Kubernetes cluster, [tcd](https://github.com/kinvolk/tcd) to do traffic control on the Kubernetes nodes and [Weave Scope](https://github.com/weaveworks/scope) to monitor, visualize and interact with the Kubernetes cluster.
 
-## What do you need to start?
+To find more information about traffic control and this demo check our [blog post](https://kinvolk.io/blog/2016/05/testing-web-services-with-traffic-control-on-kubernetes/).
+
+## Prepare for the demo
```

Comment:
There are two titles "Prepare for the demo": this one, and another one below. That's confusing.